Tasks/PublishSymbolsV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,5 +1,5 @@
 {
-  "loc.friendlyName": "Index sources and publish symbols",
+  "loc.friendlyName": "test publish symbols with linux task",
```

Comment:
Can you revert back this change. Looks like it was modified only for testing purpose?